lib/collections.js

```diff
@@ -26,12 +37,20 @@ var normalizedWords = [];
 
 // The words stored as keys of a dictionary
 var uniqueWordsAsKeys = {};
-
+  for (i = 0; i < normalizedWords.length; i++) {
+    var normWord = normalizedWords[i];
+    uniqueWordsAsKeys[normWord] = true;
+}
 // The keys pulled out into a list
-var uniqueWords = [];
 
 /* your code starts here */
-
+var uniqueWords = [];
+var j = 0;
+for (var uniqueWord in uniqueWordsAsKeys) {
+  uniqueWords[j] = uniqueWord;
+  j +=1;
```

Comment:
This will work, but in solving this particular problem, it would probably have been more idiomatic to use `.push()`.